digitalocean/Droplet.py

```diff
@@ -639,5 +640,15 @@ def get_kernel_available(self):
 
         return kernels
 
+    def update_volumes_data(self):
+        """
+            Trigger refresh of volumes table
```

Comment:
Add more description to just "trigger". What is it really doing? Why do we need to run this?
Ex: write more about what data is populating and what are the expected results by running it.

Reply:
Done